build/AudioToolbox/lib/AudioToolboxLib.vcxproj

```diff
@@ -33,10 +33,15 @@
     <ClangCompile Include="$(MSBuildThisFileDirectory)..\..\..\Frameworks\AudioToolbox\MusicSequence.mm" />
     <ClangCompile Include="$(MSBuildThisFileDirectory)..\..\..\Frameworks\AudioToolbox\MusicTrack.mm" />
     <ClangCompile Include="$(MSBuildThisFileDirectory)..\..\..\Frameworks\AudioToolbox\SystemSound.mm" />
+    <ClCompile Include="$(MSBuildThisFileDirectory)..\..\..\Frameworks\AudioToolbox\WinPhone.cpp">
+    </ClCompile>
```

Comment:
Prefer creating a separate ItemGroup for CLCompile items to make it more clear to the casual observer where the differences are.